docs/dev/zksync-v2/system-contracts.md

```diff
@@ -29,3 +29,9 @@ This contract is used to send messages from zkSync to Ethereum L1. For each mess
 [Interface](https://github.com/matter-labs/v2-testnet-contracts/blob/6a93ff85d33dfff0008624eb9777d5a07a26c55d/l2/system-contracts/interfaces/INonceHolder.sol#L5).
 
 This contract stores nonces for the accounts. The nonces for the accounts are stored in a centralized place for efficiency ([the tx nonce and the deployment nonce](./contracts.md#differences-in-create-behaviour) are stored in a single place) and also for the ease of the operator.
+
+## Bootloader
+
+For the greater extensibility and to lower the overhead, some parts of the protocol (e.g. account abstraction rules) were moved to an ephemeral contract called _bootloader_. We call it ephemeral since it is formally never deployed and it can not be called, but it has a formal [address](https://github.com/matter-labs/v2-testnet-contracts/blob/6a93ff85d33dfff0008624eb9777d5a07a26c55d/l2/system-contracts/Constants.sol#L19) that is used as `msg.sender` whenever it calls other contracts. 
+
+For now, you do not need to know any details about it, but knowing that it exists is important when you develop using the account abstraction feature. You can always assume that the bootloader is not malicious and it is a part of the protocol. In the future, the code of the bootloader will be made public and any changes to it will also mean an upgrade of the protocol.
```

Comment:
```suggestion
For now, you do not have to know any details about it, but knowing that it exists is important when you develop using the account abstraction feature. You can always assume that the bootloader is not malicious and it is a part of the protocol. In the future, the code of the bootloader will be made public and any changes to it will also mean an upgrade to the protocol.

Reply:
> You can always assume that the bootloader is not malicious

mmmm..... let's rephrase this

Reply:
"is honest"?